logstash-core/lib/logstash/api/commands/stats.rb

```diff
@@ -110,8 +130,14 @@ def plugin_stats(stats, plugin_type)
             end
           end
 
-          def report(stats)
-            {
+          def report(stats, extended_stats, opts)
```

Comment:
`PluginStats#report` is a public method, so I'd rather avoid breaking its signature; try:

```suggestion
          def report(stats, extended_stats=nil, opts={})
```